packages/core/build/tokens.ts

```diff
@@ -479,8 +479,9 @@ const aliases = {
     },
     warning: {
       value: token(color.ochre[500]),
-      tint: token(color.ochre[50]),
+      tint: token(color.ochre[100]),
       shade: token(color.ochre[600]),
+      dark: token(color.ochre[800]),
```

Comment:
To meet contrast requirements warning needs an additional darker color. For now I think it will be the exception.